### ISSUE TITLE
Show all features correctly in the search results

### ISSUE THREAD

**Issue by christophfriedrich** [Open-EO/openeo-hub#16]
In the Hub, when a user searches for backends that support certain functionalities, the Hub lists those functionalities in the result section again. Before switching to the `SupportedFeatures` component, this resulted in output like this:

> ![grafik](https://user-images.githubusercontent.com/17781682/54379394-98d3ec00-4689-11e9-9bb7-001ddf77870c.png)

I.e. functionalities not requested were simply not shown.

But now, the new component always shows *all* functionalities, and of course marks those not present in the input with a :negative_squared_cross_mark::

> ![grafik](https://user-images.githubusercontent.com/17781682/54379132-15b29600-4689-11e9-9e5b-6d821ce1685a.png)

That's not good, because these functionalities may actually be supported by the backend, but were simply not requested by the user.

I'd suggest adding a `showSupportedOnly` prop to the component that defaults to `false` but can be set to `true` to achieve what I need. Alternatively, we could add CSS classes to the `<li>` rows so that I can do `.unsupported, .partlysupported { display: none; }` in the Hub.

**Comment by christophfriedrich** [comment]
> Why would it be interesting to show what a user has already selected?

I can somewhat understand your point. The reason I did it this way is for completeness and consistency: All "categories" in the result output (collections, processes etc.) show the items that were matched within them, so I did the same with the functionalities.

You're right that at the moment it's just a repetition of the list the user already composed, but in the future we might provide more details for each functionality.

Another minor point is that (so far) there is no "You filtered with these criteria:" output, so if the user changes the form on the left-hand side, there is no record anymore of the criteria that lead to the result that is still displayed on the right-hand side -> sometimes the "repeated" information is unique.

> In addition, a user would probably be interested to see what other features are available in addition to the ones he has selected.

That is an assumption. Likewise I could argue "a user would probably be interested to see what other processes are available in addition to the ones he has selected" and thus make a case for outputting all processes the backend offers, for every search query. Obviously we're not going to do that.

> So the hub should show all endpoints and tick the supported ones. 

That would be inconsistent with the other categories, as the Hub always shows the matched items only, *not* all items. Why would we make an exception for the functionalities?

There is also an issue from the practical side, as the Hub's API doesn't return the full "supported endpoints" information (at the moment). Of course that could be changed, but semantically I'm not happy with returning the full list when the user searched for a subset of that list.
